### DISCUSSION TITLE
Increased Api Rate limits

### DISCUSSION THREAD

**Discussion by wainesiro** [ProjectOpenSea/opensea-js#1402] [Q&A]
I have a code to run api calls and create collection offers and it was working perfectly fine before today. Idk what happened but even having a delay gap of seconds between each calls the request keep getting throttled and. When it comes to speed bidding, i keep getting a error of "Exceeded Maximum Retry Limit" which was not even an issue before today. Please take a look into the updates you made. I think it is highly affecting everyone out there.

**Comment by rigwild** [comment]
I am having the exact seem issue, I opened https://github.com/ProjectOpenSea/opensea-js/issues/1408

**Comment by ryanio** [comment]
This should be resolved now. If you are looking for higher tier rate limits, please submit the form at https://docs.opensea.io/reference/api-keys#rate-limits
